packages/rum-core/src/domain/rumEventsCollection/action/trackClickActions.ts

```diff
@@ -79,11 +78,25 @@ export function trackClickActions(
   lifeCycle.subscribe(LifeCycleEventType.BEFORE_UNLOAD, stopClickChain)
   lifeCycle.subscribe(LifeCycleEventType.VIEW_ENDED, stopClickChain)
 
-  const { stop: stopActionEventsListener } = listenActionEvents({ onClick: processClick })
+  const { stop: stopActionEventsListener } = listenActionEvents<ClickActionBase>({
+    onPointerDown: (pointerDownEvent) => onPointerDown(configuration, history, pointerDownEvent),
+    onClick: (clickActionBase, clickEvent, getUserActivity) =>
+      onClick(
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏to differentiate the hook from the performed action
```suggestion
  const { stop: stopActionEventsListener } = listenActionEvents<ClickActionBase>({
    onPointerDown: (pointerDownEvent) => processPointerDown(configuration, history, pointerDownEvent),
    onClick: (clickActionBase, clickEvent, getUserActivity) =>
      processClick(
```